com.playeveryware.eos/Runtime/Core/Config/PlatformConfig.cs

```diff
@@ -51,28 +71,430 @@ public abstract class PlatformConfig : Config
 #if !EOS_DISABLE
+        /// <summary>
+        /// Flags; used to initialize the EOS platform.
+        /// </summary>
+        [ConfigField("Platform Flags",
+            ConfigFieldType.Enum,
+            "Platform option flags",
+            1, "https://dev.epicgames.com/docs/epic-online-services/eos-get-started/working-with-the-eos-sdk/eos-overlay-overview#eos-platform-flags-for-the-eos-overlay")]
+        [JsonConverter(typeof(ListOfStringsToPlatformFlags))]
+        public WrappedPlatformFlags platformOptionsFlags;
+
+        /// <summary>
+        /// Flags; used to set user auth when logging in.
+        /// </summary>
+        [ConfigField("Auth Scope Flags",
+            ConfigFieldType.Enum,
+            "Platform option flags",
+            1, "https://dev.epicgames.com/docs/api-ref/enums/eos-e-auth-scope-flags?lang=en-US")]
+        [JsonConverter(typeof(ListOfStringsToAuthScopeFlags))]
+        public AuthScopeFlags authScopeOptionsFlags;
+
         /// <summary>
         /// Used to store integrated platform management flags.
         /// </summary>
+        [ConfigField("Integrated Platform Management Flags", 
+            ConfigFieldType.Enum, "Integrated Platform Management " +
+                                  "Flags for platform specific options.",
+            1, "https://dev.epicgames.com/docs/api-ref/enums/eos-e-integrated-platform-management-flags")]
         [JsonConverter(typeof(ListOfStringsToIntegratedPlatformManagementFlags))]
-        public IntegratedPlatformManagementFlags flags;
+        [JsonProperty("flags")] // Allow deserialization from old field member.
+        public IntegratedPlatformManagementFlags integratedPlatformManagementFlags;
 #endif
 
+        #endregion
+
+        #region Thread Affinity & Various Time Budgets
+
+        /// <summary>
+        /// Tick Budget; used to define the maximum amount of execution time the
+        /// EOS SDK can use each frame.
+        /// </summary>
+        [ConfigField("Tick Budget (ms)",
+            ConfigFieldType.Uint,
+            "Used to define the maximum amount of execution time the " +
+            "EOS SDK can use each frame.",
+            2)]
+        public uint tickBudgetInMilliseconds;
+
+        /// <summary>
+        /// TaskNetworkTimeoutSeconds; used to define the maximum number of
+        /// seconds the EOS SDK will allow network calls to run before failing
+        /// with EOS_TimedOut. This plugin treats any value that is less than or
+        /// equal to zero as using the default value for the EOS SDK, which is
+        /// 30 seconds.
+        ///
+        /// This value is only used when the <see cref="NetworkStatus"/> is not
+        /// <see cref="NetworkStatus.Online"/>.
+        /// <seealso cref="PlatformInterface.GetNetworkStatus"/>
+        /// </summary>
+        [ConfigField("Network Timeout Seconds",
+            ConfigFieldType.Double,
+            "Indicates the maximum number of seconds that EOS SDK " +
```

Comment:
This is missing a really important piece of context. This timeout only applies *while EOS is not online yet*. It doesn't cover calls after becoming online.

https://dev.epicgames.com/docs/api-ref/structs/eos-platform-options

Reply:
Good note - I've changed it to make it much clearer.